autogen/agentchat/contrib/agent_builder.py

```diff
@@ -203,7 +203,7 @@ def __init__(
             builder_filter_dict.update({"model": builder_model})
         if len(builder_model_tags) != 0:
             builder_filter_dict.update({"tags": builder_model_tags})
-        builder_config_list = autogen.config_list_from_json(config_file_or_env, filter_dict=builder_filter_dict)
+        builder_config_list = autogen.config_list_from_json(config_file_or_env, file_location=config_file_location, filter_dict=builder_filter_dict)
```

Comment:
Need to fix the code formatting to pass the test.
```suggestion
        builder_config_list = autogen.config_list_from_json(
            config_file_or_env, file_location=config_file_location, filter_dict=builder_filter_dict
        )
```

Reply:
fixed.